### DISCUSSION TITLE
Redis.call() function showing error

### DISCUSSION THREAD

**Discussion by sreerajseeroo** [adonisjs/core#3962] [Help] [answered]
let postjson1 = await Redis.call("JSON.SET", "postJson",'{"name":"xyxz"}', (err, value) => {
    // value === 'bar'
    console.log(err,"Redis eror",value)
  });

This is the sample code for saving JSON. While running the code I got some errors like below. 
 
TypeError: Redis_1.default.call is not a function. 

 Any Idea for the error? How it will resolve?

Adonis Js 5  with Redis packages
Node 18. x

**Comment by RomainLanz** [comment, marked as answer]
Hey @sreerajseeroo! 👋🏻 

The `call` method has not been released on our package yet.

**Reply by sreerajseeroo** [reply]
Thanks @RomainLanz  . any other solution for running JSON.SET command from adonis js

**Reply by RomainLanz** [reply]
I have just published a released. Can you try to update `@adonisjs/redis`?

**Reply by sreerajseeroo** [reply]
Okay, let me check

**Reply by sreerajseeroo** [reply]
await Redis.call("JSON.SET","example1",'$','{"name":"XYZ"}') .
 Now executed like this, it will work, But will get an argument mismatch error while executing the command without the $ symbol. I think the $ symbol is mandatory. right?
Thanks for your quick support
